src/Microsoft.ServiceFabric.Actors/Runtime/ActorService.cs

```diff
@@ -60,15 +60,15 @@ public class ActorService : StatefulServiceBase, IActorService
             this.stateProvider = (IActorStateProvider) this.StateProviderReplica;
             this.settings = ActorServiceSettings.DeepCopyFromOrDefaultOnNull(settings);
 
-            // Set internal components
+            // Sets internal components.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.